src/WebJobs.Script.WebHost/Diagnostics/SystemLogger.cs

```diff
@@ -173,9 +176,28 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
                     functionName = string.IsNullOrEmpty(fex.MethodName) ? string.Empty : fex.MethodName.Replace("Host.Functions.", string.Empty);
                 }
 
-                (innerExceptionType, innerExceptionMessage, details) = exception.GetExceptionDetails();
-                formattedMessage = Sanitizer.Sanitize(formattedMessage);
-                innerExceptionMessage = innerExceptionMessage ?? string.Empty;
+                var baseEx = exception.GetBaseException();
+                innerExceptionType = baseEx.GetType().FullName;
+
+                var originalMessage = baseEx.Message;
+                var formattedDetails = exception.ToFormattedString();
+
+                const string RedactedMessage = "An exception occurred during invocation, but its details are redacted. Customers with AppInsights or OTel enabled can access full exception details.";
```

Comment:
Wondering if we can't just simplify this to "[Redacted]". Not sure we need all this log bloat.

Reply:
I can, but I'm aiming to make it clear that we have the option to request these error messages from customers if needed for investigation.
I can change it to "Redacted - customers using AppInsights or OTel can view full details." to make it short.